templates/template_Beamer.tex

```diff
@@ -7,6 +7,7 @@
 \begin{frame}[plain]
     \maketitle
 \end{frame}
-\begin{frame}{Frame Title}
+\begin{frame}
+\frametitle{Frame Title}
```

Comment:
Apply some indent, just like the `\maketitle` on line 8?
```suggestion
    \frametitle{Frame Title}
```
(Didn't check if 4-space indent is uniformly applied in all templates.)

Reply:
done